content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Install the AWS Integration
+
+As a first step, make sure to [install the AWS integration][1] before proceeding.
+
+Next, depending on your language and configuration, choose between using the Datadog Lambda Library or Amazon X-Ray for your traces.
+
+{{< img src="tracing/serverless_functions/LambdaLibraryXRay.png" alt="Lambda Library or X-Ray"  style="width:100%;">}}
+
+## Use the Datadog Lambda Library
```

Comment:
```suggestion
## Use the Datadog APM
```

Reply:
I also think we should be more explicit in this section in being like "these docs live elsewhere"

Reply:
In general, on the UX side this section doesn't feel well highlighted, and really easy to skip over.